exp-android-sdk/src/main/java/com/scala/exp/android/sdk/Runtime.java

```diff
@@ -166,6 +171,75 @@ public void call(Throwable throwable) {
                 });
     }
 
+
+    /**
+     * Start SDK with Auth
+     * @param auth
+     * @return
+     */
+    public static Observable<Boolean> start_auth(final Auth auth){
```

Comment:
Try to combine this into the other start_auth() method, they seem to share a lot of code